### PR TITLE
Connect PSACoverCropCouncil to selector #700

### PR DESCRIPTION
**Pull request by ravighevariya10**
Created a shared CoverCropCouncil logo displayer

### CODE REVIEW

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://,51.8.10.174,:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://51.8.10.174:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Review by YazidZidane — COMMENTED** [review]
Can you also move the `useEffect()` logic on line 87 and 105 into the custom component? The first one is for setting favicon and the second one is for setting the image (you might need to add a prop `council`).

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://51.8.10.174:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://51.8.10.174:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://,51.8.10.174:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

File: public/images/neccc_logo.png
Comment (Adamws33) [inline public/images/neccc_logo.png:1]: I don't believe we need this.  All of our logos come from the API

**Review by YazidZidane — APPROVED** [review]

**Pull request merged**